src/usr/local/www/system_camanager.php

```diff
@@ -308,6 +317,7 @@
 
 		if (!$input_errors) {
 			write_config();
+			ca_trustedcerts_generate();
```

Comment:
@NOYB Except that it does offer a benefit. The benefit is you know 100% that the file contents match what is in config.xml. The ultimate reference is config.xml. The contents of config.xml must always be assumed to be correct, and is one of the basic principles of pfSense. If that is wrong then you need to fix what broke config.xml.

A user must be able to take config.xml and restore it to another system and end up with the same consistent system. Though packages offer some exceptions in that area, the base system has to adhere to that principle.